oboe-sys/build.rs

```diff
@@ -127,8 +127,19 @@ fn select_library(prebuilt_dir: &Path) -> Vec<LinkArg> {
 
     let lib_name = "oboe-ext".into();
 
+    let source_so = prebuilt_dir.join(lib_arch).join("liboboe-ext.so");
+
+    let out_dir = PathBuf::from(
+        env::var("OUT_DIR").expect("OUT_DIR is set by cargo.")
+    );
+    let target_so = out_dir
+        .parent().unwrap()
+        .parent().unwrap()
+        .parent().unwrap()
+        .join("deps")
+        .join("liboboe-ext.so");
+    std::fs::copy(source_so, target_so).expect("Failed to copy liboboe-ext.so");
```

Comment:
At least you should copy shared libs once....

Actually I think in that case copying is not right thing because that directory is for Rust output not for third dylibs. Maybe at most as temporary fix to get things works. But actually we just need to fix _cargo-apk_. I in progress and fix comming soon (I hope).

Reply:
Yeah this is intended as a temporary fix until there is a better way.

Reply:
It needs only in case of static linking which currently does not works proper.